src/index.css

```diff
@@ -1,5 +1,12 @@
+@import "~@blueprintjs/core/lib/css/blueprint.css";
+@import "~@blueprintjs/icons/lib/css/blueprint-icons.css";
+
 body {
   margin: 0;
   padding: 0;
+  padding-top: 50px;
   font-family: sans-serif;
 }
+.App {
+  text-align: center;
+}
```

Comment:
with `styled-components` would be more consistent to do it with `injectGlobal` (in v4 they another approach)